apps/web/app/(app)/gardens/[chain]/[garden]/pool/[poolId]/page.tsx

```diff
@@ -60,101 +72,89 @@ export default async function Pool({
   );
 
   return (
-    <div className="relative mx-auto flex max-w-7xl gap-3 px-4 sm:px-6 lg:px-8">
-      <div className="flex flex-1 flex-col gap-6 rounded-xl border-2 border-black bg-surface p-16">
-        <header className="flex flex-col items-center justify-center">
-          <h2 className="text-center font-press">Pool {poolId} </h2>
+    <div className="mx-auto flex max-w-5xl flex-col gap-10">
+      {/* Header */}
+      <section className="section-layout flex flex-col gap-0 overflow-hidden">
+        <header>
+          <h2>
+            Pool #{poolId} - {title}
```

Comment:
I think we should go for h2 pool title and #{poolId}  much smaller, could be h4 or h5

Reply:
I this case, we should ask Salvador to provide in designs and then check how it looks before we implement anything